TEDU-54 Reorganized Campaigns

swati-thacker13 opened the pull request:
As part of GSOD, reorganized campaigns as follows:

1. Campaigns (Introduction)
2. Creating a Campaign
3. Using the Campaign Builder
 - Actions
 - Decision
 - Conditions
4. Managing Campaigns
5. Troubleshooting Campaigns

RCheesley commented on pages/05.Campaigns/docs.en.md:
```diff
@@ -7,25 +7,27 @@ taxonomy:
-### Campaign Automation
-
-One of the main benefits of this campaign workflow process is the ability to predefine these workflows and have them respond automatically to your contacts and timelines. This automation minimizes the amount of time required for manual contact activity and improves reliability of contact nurturing through consistent contact.
+> **Note**: Campaigns can trigger a variety of actions. Email actions are mentioned in the above campaign types only as an example. The other actions that a campaign can trigger include automatic assignment to a new segment, assigning a new point value, or an integration into a CRM or other systems.
```
```suggestion
>>> Campaigns can trigger a variety of actions. Email actions are mentioned in the above Campaign types only as an example. The other actions that a Campaign can trigger include automatic assignment to a new Segment, assigning a new Point value, or pushing into an integration into a CRM or other systems.
```

Maybe consider linking this to the docs on Actions?

swati-thacker13 replied:
About linking this to docs on Actions, it will be breaking the flow considering that we introduce the concept of Actions, Decisions, and Conditions only in the next chapter.

swati-thacker13 commented on pages/05.Campaigns/01.Creating Campaigns/docs.en.md:
```diff
@@ -0,0 +1,79 @@
+
+# Creating campaigns
+
+Creating campaigns is a simple process. When you create a new campaign, you perform the basic administrative tasks such as, choosing a name for the campaign, creating a description, assigning a category, and define publishing information for the campaign.
```
Agree. Marketing Automation is not a service that is exclusive to Mautic. It's an industry-standard term. Therefore, removing the capitalization.

swati-thacker13 commented on pages/05.Campaigns/01.Creating Campaigns/docs.en.md:
```diff
@@ -0,0 +1,79 @@
+
+# Creating campaigns
+
+Creating campaigns is a simple process. When you create a new campaign, you perform the basic administrative tasks such as, choosing a name for the campaign, creating a description, assigning a category, and define publishing information for the campaign.
```
Agree. Committed the change.

swati-thacker13 commented on pages/05.Campaigns/01.Creating Campaigns/docs.en.md:
```diff
@@ -0,0 +1,79 @@
+
+---------------------
+
+# Creating campaigns
```
Agree. Committed. Changed it throughout the Campaigns chapter.

swati-thacker13 commented on pages/05.Campaigns/02. Using Campaign Builder/01.Actions/docs.en.md:
```diff
@@ -0,0 +1,67 @@
+---------------------
+## Actions
+
+Campaign actions are events that you initiate on your contacts or contact records. These can represent sending communications to the contact or automate operational tasks to keep your marketing running. A single campaign can include more than one action. When you create a campaign, you select one of these actions to begin the workflow.
```
Removed the comma. Commas are used with lists in 'and' and 'or' when there are more than two items in a list.

swati-thacker13 commented on pages/05.Campaigns/02. Using Campaign Builder/01.Actions/docs.en.md:
```diff
@@ -0,0 +1,67 @@
+
+| Action        | Description  | 
+| :------------- | :----------: | 
+| **Add Do Not Contact**|Adds the user to the <br>do not contact list |
```
Understood and agree.

swati-thacker13 commented on pages/05.Campaigns/02. Using Campaign Builder/01.Actions/docs.en.md:
```diff
@@ -0,0 +1,67 @@
+|**Remove Do Not Contact**| Removes the user from the <br> do not contact list.|
+|**Send a webhook**| Sends a webhook to a defined URL,<br> using the GET, POST, PUT, PATCH, or DELETE methods. Headers and data are customizable, and support the use of tokens, such as contact field tokens and contact's IP address as a token. For example, {contactfield=firstname}|
+|**Send email**| Sends a Transaction or Marketing email <br> to the selected contact. A transactional <br> email can be sent to the contact multiple <br> times. A marketing email can be sent <br> to the contact only once across multiple sources. If the contact has already <br> received this email from another source or <br> the current campaign, the email will not be <br> sent again and the conta ...
+|**Send email to user**| Sends an email to an entity other <br> than the contact. This may be a Mautic <br> user, the contact’s owner, or non-users.<br> Emails sent using this action does not <br> generate any statistics for contacts <br> or emails.|
```
The UI terms should appear as it is (with the same case) in the docs. For example, **Send email to user** not **Send Email to user**.

swati-thacker13 commented on pages/05.Campaigns/02. Using Campaign Builder/01.Actions/docs.en.md:
```diff
@@ -0,0 +1,67 @@
+|**Send a webhook**| Sends a webhook to a defined URL,<br> using the GET, POST, PUT, PATCH, or DELETE methods. Headers and data are customizable, and support the use of tokens, such as contact field tokens and contact's IP address as a token. For example, {contactfield=firstname}|
+|**Send email**| Sends a Transaction or Marketing email <br> to the selected contact. A transactional <br> email can be sent to the contact multiple <br> times. A marketing email can be sent <br> to the contact only once across multiple sources. If the contact has already <br> received this email from another source or <br> the current campaign, the email will not be <br> sent again and the conta ...
+|**Send email to user**| Sends an email to an entity other <br> than the contact. This may be a Mautic <br> user, the contact’s owner, or non-users.<br> Emails sent using this action does not <br> generate any statistics for contacts <br> or emails.|
+|**Send marketing message**| Sends a message using the contact's <br> prefered channel.|
```
The UI terms should appear as it is (with the same case) in the docs.

swati-thacker13 commented on pages/05.Campaigns/02. Using Campaign Builder/01.Actions/docs.en.md:
```diff
@@ -0,0 +1,67 @@
+|**Send email**| Sends a Transaction or Marketing email <br> to the selected contact. A transactional <br> email can be sent to the contact multiple <br> times. A marketing email can be sent <br> to the contact only once across multiple sources. If the contact has already <br> received this email from another source or <br> the current campaign, the email will not be <br> sent again and the conta ...
+|**Send email to user**| Sends an email to an entity other <br> than the contact. This may be a Mautic <br> user, the contact’s owner, or non-users.<br> Emails sent using this action does not <br> generate any statistics for contacts <br> or emails.|
+|**Send marketing message**| Sends a message using the contact's <br> prefered channel.|
+|**Update contact**| Updates the existing contact's fields <br> with the specified values.|
```
The UI terms should appear as it is (with the same case) in the docs.

swati-thacker13 commented on pages/05.Campaigns/02. Using Campaign Builder/01.Actions/docs.en.md:
```diff
@@ -0,0 +1,67 @@
+|**Send email to user**| Sends an email to an entity other <br> than the contact. This may be a Mautic <br> user, the contact’s owner, or non-users.<br> Emails sent using this action does not <br> generate any statistics for contacts <br> or emails.|
+|**Send marketing message**| Sends a message using the contact's <br> prefered channel.|
+|**Update contact**| Updates the existing contact's fields <br> with the specified values.|
+|**Update contact's primary company**| Updates the existing contact's <br> primary company fields with the <br> specified value. See <br> [Mautic's companies support][companies].|
```
The UI terms should appear as it is (with the same case) in the docs.

swati-thacker13 commented on pages/05.Campaigns/02. Using Campaign Builder/01.Actions/docs.en.md:
```diff
@@ -0,0 +1,67 @@
+|**Send marketing message**| Sends a message using the contact's <br> prefered channel.|
+|**Update contact**| Updates the existing contact's fields <br> with the specified values.|
+|**Update contact's primary company**| Updates the existing contact's <br> primary company fields with the <br> specified value. See <br> [Mautic's companies support][companies].|
+|**Update contact owner**| Updates the contact's owner.|
```
The UI terms should appear as it is (with the same case) in the docs.

swati-thacker13 commented:
@RCheesley. Thank you for your detailed review of this mammoth PR!

I am in the process of addressing your suggestions. However, I have a couple of observations on your comment about capitalization:

- [Ruth] When we are referring to a component or feature of Mautic, we should capitalise the name of it: https://developers.google.com/style/capitalization?hl=en
Now, where this gets tricky is that we could be referring to the component/feature (eg Contact, Email) but equally to the action (send an Email, add the Contact).

I have capitalised these, but am open to discussion on whether they should indeed be capitalised. The most important part here is to be consistent - the linter will go some way to help with that!

 - [Swati] While I agree with your suggestion about capitalizing references to different features of Mautic, I don't quite agree that we should capitalize any action associated with it, such as sending an **e**mail, adding a **c**ontact, etc. This is mostly because too much capitalization in paragraphs and sentences thwarts readability and makes for a difficult reading. It's not soothing to the eyes. 

That being said, we will have to define clear rules for when to capitalise the feature names and when not. 

Let me know what you think.

swati-thacker13 commented on pages/05.Campaigns/02. Using Campaign Builder/02.Decisions/docs.en.md:
```diff
@@ -0,0 +1,67 @@
+
+| Decision        | Description  | 
+| :------------- | :----------: |
+|**Opens email**| Tracks whether the contact <br> opens the email.|
```
UI terms should appear as it is (with the same case) in the docs.

swati-thacker13 commented on pages/05.Campaigns/02. Using Campaign Builder/02.Decisions/docs.en.md:
```diff
@@ -0,0 +1,67 @@
+| Decision        | Description  | 
+| :------------- | :----------: |
+|**Opens email**| Tracks whether the contact <br> opens the email.|
+|**Clicks email**|Tracks whether the contact <br> clicks a link within the send email.|
```
UI terms should appear as it is (with the same case) in the docs.

swati-thacker13 commented on pages/05.Campaigns/02. Using Campaign Builder/02.Decisions/docs.en.md:
```diff
@@ -0,0 +1,67 @@
+| :------------- | :----------: |
+|**Opens email**| Tracks whether the contact <br> opens the email.|
+|**Clicks email**|Tracks whether the contact <br> clicks a link within the send email.|
+|**Replies to email**|Tracks if a contact has replied to an<br> email that you sent.|
```
UI terms should appear as it is (with the same case) in the docs.

swati-thacker13 commented on pages/05.Campaigns/02. Using Campaign Builder/02.Decisions/docs.en.md:
```diff
@@ -0,0 +1,67 @@
+
+
+> **Note**:
+A contact must already be part of the campaign in order for it to recognize the decision that is executed. Therefore, campaigns should never start with a decision unless you are manually managing the contacts assigned to it and the decision is expected to be executed at a later time.
```
I don't understand this bit either. 'Therefore, campaigns should never......at a later time.' 

I have retained the note as it was. What should we change this to?

RCheesley replied:
I would be inclined to remove it as it is confusing. We could always add it back in later with more clarity!

swati-thacker13 replied:
Removed this.

swati-thacker13 commented on pages/05.Campaigns/02. Using Campaign Builder/03.Conditions/docs.en.md:
```diff
@@ -0,0 +1,63 @@
+
+ | Condition        | Description  | 
+| :------------- | :----------: |
+|**Contact campaigns**| Checks if the contact is a member of another campaign|
```
UI terms should appear as it is (with the same case) in the docs.

swati-thacker13 commented on pages/05.Campaigns/02. Using Campaign Builder/03.Conditions/docs.en.md:
```diff
@@ -0,0 +1,63 @@
+|**Contact device**|Checks if the contact has interacted <br> with your campaign from a specific device <br>type, brand, or OS|
+|**Contact field value**| Checks if the information matches the selected criteria on the contact record, the contact’s primary company, or UTM tags|
+|**Contact owner**| Checks if the selected user is assigned <br> as the contact’s owner|
+|**Contact segments**| Checks if the contact is a member of <br>selected segments|
```
UI terms should appear as it is (with the same case) in the docs.

swati-thacker13 commented on pages/05.Campaigns/02. Using Campaign Builder/03.Conditions/docs.en.md:
```diff
@@ -0,0 +1,63 @@
+|**Contact field value**| Checks if the information matches the selected criteria on the contact record, the contact’s primary company, or UTM tags|
+|**Contact owner**| Checks if the selected user is assigned <br> as the contact’s owner|
+|**Contact segments**| Checks if the contact is a member of <br>selected segments|
+|**Contact tags**|Checks if specified tags are on the contact record|
```
UI terms should appear as it is (with the same case) in the docs.

swati-thacker13 commented on pages/05.Campaigns/02. Using Campaign Builder/03.Conditions/docs.en.md:
```diff
@@ -0,0 +1,63 @@
+|**Has active notification**|Checks if an active web notification <br> is being sent to the contact|
+|**Has valid email address**|Checks if the contact’s email address has a <br>valid syntax, that is, name@domain.com without spaces, other invalid characters, formats.|
+|
+The delay you set is ran before checking the condition no matter the delay you add on the connected actions. It will not wait the delay on the connected action to check the status of the condition to qualify the contact into the positive or negative path of the condition. <This sounds very convoluted. Need to understand and rewrite.>
```
Thanks, Ruth. I updated your suggestion a bit for syntax issues.

swati-thacker13 commented on pages/05.Campaigns/02. Using Campaign Builder/docs.en.md:
```diff
@@ -0,0 +1,57 @@
+
+ - **Contact forms**: Choose this option if you want to use forms that your contacts have intercted with. Forms are the primary point of customer contact and are used to gather information about a contact. This information can then be used to perform a number of actions in a campaign. 
+ 
+   You can select a mix of both types of contact sources for your campaign. To use both, use the grey selector button on either the left or right side of the contact source box to add whichever source type you didn’t originally select.
```
We don't use 'click on'. The action of clicking includes the preposition on.

swati-thacker13 commented on pages/05.Campaigns/03. Managing Campaigns/docs.en.md:
```diff
@@ -0,0 +1,46 @@
+
+The **Conditions** tab displays a tabular list of all he conditions that you have added to your campaign.
+
+The **Contacts** tab displays a grid view of all the contacts that you have added to your campaign. 
```
The norm in tech writing is to use active voice. 'been added' is passive voice. So, changing it back to active voice.

swati-thacker13 commented on pages/05.Campaigns/docs.en.md:
```diff
@@ -7,25 +7,27 @@ taxonomy:
+A campaign is a marketing activity that aligns Components and the Channels through which you can publish your content in a coordinated, strategically timed approach to meet specific business objectives. After you add your contacts and set up the required channels, you can create campaigns to build meaningful relationships with your contacts.
 
-Campaigns are central to creating an automated workflow to assist with your marketing efforts. Campaigns consist of various external points of contact which will engage your contacts. These engagements can be created to occur on predefined time intervals or in response to specific contact actions.
+Campaigns are useful for contact management, marketing operations, and sales enablement. A campaign can send tailored messages to all users in the segment, or to a subset of users that you specify. You can set the campaign schedule to send the message once or at a recurring interval, such as once a week.
```
I agree.

RCheesley commented:
@Swati-Thacker looks like one conflicting file to address then we can get this merged!

swati-thacker13 commented on pages/05.Campaigns/01.Creating Campaigns/docs.en.md:
```diff
@@ -0,0 +1,79 @@
+## Prerequisites
+
+Before you start creating campaigns, you must ensure that you have the following set up:
+<!-- What are the prerequisites of campaign creation? Need more information in this section-->
```
Revamped the Prerequisites section.

swati-thacker13 commented on pages/05.Campaigns/01.Creating Campaigns/docs.en.md:
```diff
@@ -0,0 +1,79 @@
+    See documentation on [cron jobs][cron-jobs] for further details.
+
+
+2. Create components
```
Added examples of components.

swati-thacker13 commented on pages/05.Campaigns/01.Creating Campaigns/docs.en.md:
```diff
@@ -0,0 +1,79 @@
+
+
+2. Create components
+3. Set up the channels
```
Added examples of Channels and moved Channels above Components.

swati-thacker13 commented:
Hi @RCheesley. I addressed all your comments and resolved them. Based on your comments, made the following further changes:

- Revamped the **Prerequisites** section of the Creating Campaigns page.
- Renamed ‘Executing Campaign Actions’ to ‘Trigger campaign events’ and moved it to the Using the Campaign builder page.
- From the **Replies to email** decision column on the Decision page, added a link to Contact replies.
- Checked and removed remaining line breaks from tables.
- Checked all images for alt text.
- Added links to Conditions, Decisions, and Actions from the Managing Campaigns page.
- Capitalized all instances of Mautic related terms.

RCheesley commented:
Great, @mohit-rocks will be able to help with addressing the conflict then we should be good to go!

RCheesley commented on pages/06.Campaigns/02. Using Campaign Builder/.revs/20200306-151901/docs.en.md:
```diff
@@ -0,0 +1,98 @@
+
+The first thing to be selected is where the campaign will pull in contacts from or contact sources. There are currently two options for contact sources: segments and forms. One or both can be added to the campaign.
+ 
+![](contact-sources.png)
```
```suggestion
![Screenshot showing contact sources](contact-sources.png)
```

RCheesley commented on pages/06.Campaigns/02. Using Campaign Builder/.revs/20200306-151901/docs.en.md:
```diff
@@ -0,0 +1,98 @@
+
+After selecting one or more sources, the next step will be to add one or more actions (most likely), decisions and/or conditions:
+
+![](events.png)
```
```suggestion
![Screenshot showing actions](events.png)
```

swati-thacker13 replied:
This comment was on an older file that was merged. This file doesn't exist any longer.

RCheesley commented on pages/06.Campaigns/02. Using Campaign Builder/.revs/20200306-151901/docs.en.md:
```diff
@@ -0,0 +1,98 @@
+
+When you create a campaign you will select one of these actions to begin the workflow. In most cases this initial step will be an email sent to your segments.
+
+![](send-email-delay.png)
```
```suggestion
![Screenshot showing email delay](send-email-delay.png)
```

RCheesley commented on pages/06.Campaigns/02. Using Campaign Builder/.revs/20200306-151901/docs.en.md:
```diff
@@ -0,0 +1,98 @@
+
+You will notice that when you add an email to a campaign you will be able to select a potential *delay* for when the email is delivered. If the action is attached to a decision's non-action initiated decision path, the delay becomes how long the contact has to take action before the campaign progresses down the non-action path. 
+
+![](send-email-delay-nonaction.png)
```
```suggestion
![Screenshot showing delay on non action](send-email-delay-nonaction.png)
```

swati-thacker13 replied:
This comment was on an older file that was merged. This file doesn't exist any longer.

RCheesley commented on pages/06.Campaigns/02. Using Campaign Builder/.revs/20200306-151901/docs.en.md:
```diff
@@ -0,0 +1,98 @@
+
+Decisions are taken in response to an action and as such a decision has two outcomes.
+
+![](campaign-decisions.gif)
```
```suggestion
![Screenshot showing campaign decisions](campaign-decisions.gif)
```

swati-thacker13 replied:
This comment was on an older file that was merged. This file doesn't exist any longer.

RCheesley commented on pages/06.Campaigns/02. Using Campaign Builder/.revs/20200306-151901/docs.en.md:
```diff
@@ -0,0 +1,98 @@
+
+These two options are demonstrated by the green and red decision points on the decision. Each path can then be handled by your campaign. This process is typically referred to as a **decision tree**.
+
+__It is important to note that a contact must already be part of the campaign in order for it to recognize the decision executed. Therefore, campaigns should never start with a decision unless you are manually managing the contacts assigned to it and the decision is expected to be executed at a later time.__ 
```
```suggestion
>>> It is important to note that a contact must already be part of the campaign in order for it to recognize the decision executed. Therefore, campaigns should never start with a decision unless you are manually managing the contacts assigned to it and the decision is expected to be executed at a later time.
```

swati-thacker13 replied:
This comment was on an older file that was merged. This file doesn't exist any longer.

RCheesley commented on pages/06.Campaigns/02. Using Campaign Builder/.revs/20200306-151901/docs.en.md:
```diff
@@ -0,0 +1,98 @@
+#### Example
+
+To provide a simple example of a decision tree consider an email where the decision is to open an email. There are two outcomes, if the contact chooses to open the email then the green decision point connects to the next action to be taken in the campaign workflow. If, however, the contact does not open the email then you may desire a different action to be taken (e.g. a delay of 30 days then a s ...
+
```
```suggestion
```

swati-thacker13 replied:
This comment was on an older file that was merged. This file doesn't exist any longer.

RCheesley commented on pages/06.Campaigns/01.Creating Campaigns/.revs/20200615-225435/docs.en.md:
```diff
@@ -0,0 +1,59 @@
+
+The basics of campaign creation are handled easily by the initial screen but the finer details of building a campaign occur within the campaign builder. This might be considered advanced campaign creation but every campaign does need to use the campaign builder.
+
+
```
```suggestion
```

swati-thacker13 replied:
This comment was on an older file that was merged. This file doesn't exist any longer.

RCheesley commented on pages/06.Campaigns/02. Using Campaign Builder/01.Actions/docs.en.md:
```diff
@@ -0,0 +1,68 @@
+2. The Delete Contact action also deletes the Campaign event log record about that contact. Therefore, though this action might always display 0% progress on the Campaign detail page, it could have deleted some Contacts.
+
+   The Delete Contact action doesn't allow other Campaign events to be connected to it. Since the Contact will not exist after this action is triggered, Campaign events cannot be triggered after this point.
+
```
```suggestion
```

swati-thacker13 replied:
This comment was on an older file that was merged. This file doesn't exist any longer.

RCheesley commented on pages/06.Campaigns/02. Using Campaign Builder/01.Actions/docs.en.md:
```diff
@@ -0,0 +1,68 @@
+
+   The Delete Contact action doesn't allow other Campaign events to be connected to it. Since the Contact will not exist after this action is triggered, Campaign events cannot be triggered after this point.
+
+
```
```suggestion
```

swati-thacker13 replied:
Done!

RCheesley commented on pages/06.Campaigns/02. Using Campaign Builder/02.Decisions/docs.en.md:
```diff
@@ -0,0 +1,62 @@
+
+Depending on whether the criteria for the decision is met, the Contact is sent down either the green or the red paths in the decision tree. For example, consider an instance where the decision is to open an email. There can be two outcomes. If the Contact chooses to open the email, then the green decision path connects to the next action to be taken in the Campaign workflow. If, however, the Cont ...
+
+
```
```suggestion
```

swati-thacker13 replied:
Done!

RCheesley commented on pages/06.Campaigns/02. Using Campaign Builder/02.Decisions/docs.en.md:
```diff
@@ -0,0 +1,62 @@
+Depending on whether the criteria for the decision is met, the Contact is sent down either the green or the red paths in the decision tree. For example, consider an instance where the decision is to open an email. There can be two outcomes. If the Contact chooses to open the email, then the green decision path connects to the next action to be taken in the Campaign workflow. If, however, the Cont ...
+
+
+
```
```suggestion
```

swati-thacker13 replied:
Done.

RCheesley commented on pages/06.Campaigns/02. Using Campaign Builder/02.Decisions/docs.en.md:
```diff
@@ -0,0 +1,62 @@
+|**Visits a page**|Specify one or multiple Pages you want the Contact to visit. Can be Mautic Landing Pages or pages on your website.|
+|
+
+<br>
```
```suggestion
```

swati-thacker13 replied:
Done.

RCheesley commented on pages/06.Campaigns/02. Using Campaign Builder/02.Decisions/docs.en.md:
```diff
@@ -0,0 +1,62 @@
+|**Clicks email**|Tracks whether the Contact clicks a link within the sent Email. This infers that the email has been opened.|
+|**Replies to email**|Tracks if a Contact has replied to an Email that you sent. For more information, see [Contact replies][contact-replies]. |
+|
+<br>
```
```suggestion
```

swati-thacker13 replied:
Done

RCheesley commented on pages/06.Campaigns/02. Using Campaign Builder/02.Decisions/docs.en.md:
```diff
@@ -0,0 +1,62 @@
+|**Replies to email**|Tracks if a Contact has replied to an Email that you sent. For more information, see [Contact replies][contact-replies]. |
+|
+<br>
+<Link here>
```
```suggestion
```

swati-thacker13 replied:
Done

RCheesley reviewed (changes requested):
Some missing image alt text and some extra lines that I don't think are needed!

Otherwise good to go!

RCheesley commented on pages/06.Campaigns/02. Using Campaign Builder/01.Actions/docs.en.md:
```diff
@@ -0,0 +1,66 @@
+1.  As the first step of your Campaign, you typically send out an Email to your Segments. When you add an Email to a Campaign, you can select a potential **delay** for when the Email is delivered as shown in the following image.
+
+ If the action is attached to a decision's **non-action** initiated decision path, the delay becomes how long the Contact has to take action before the Campaign progresses down the non-action path. 
+<br>
```
```suggestion

```

RCheesley commented on pages/06.Campaigns/02. Using Campaign Builder/docs.en.md:
```diff
@@ -0,0 +1,65 @@
+---------------------
+The Campaign Builder provides a blank canvas on which you can build your Campaign workflow. A Campaign Builder is made up of conditions, decisions, and actions. It  enables you to create a simple workflow by dragging and dropping various decisions, actions, and conditions onto a canvas.
+
+
```
```suggestion
```

RCheesley commented on pages/06.Campaigns/02. Using Campaign Builder/docs.en.md:
```diff
@@ -0,0 +1,65 @@
+
+3. After selecting one or more Contact sources, click the grey selector button to add at least one event to your Campaign. A Campaign event comprises of a combination of actions, decisions, and/or conditions as shown in the following image:
+
+
```
```suggestion
```

RCheesley commented on pages/06.Campaigns/03. Managing Campaigns/docs.en.md:
```diff
@@ -0,0 +1,50 @@
+
+The **Recent Activity** panel on the right displays the recent activities that take place in the Campaign.
+
+<Links here>
```
```suggestion
```

RCheesley reviewed (changes requested):
Tiny suggestions and then we are good to merge!

RCheesley reviewed (approved):
LGTM ! 🚀
